### ISSUE TITLE
App crashes when saving empty PGP Key

### ISSUE THREAD

**Issue by ttyz** [mssun/passforios#26]
After saving an empty PGP key, the application crashes:

Debug info:
```
2017-02-22 13:46:28.102 pass[4066:71397] *** Assertion failure in -[ObjectivePGP keysFromData:], /Users/tyz/src/pass-ios/Pods/ObjectivePGP/ObjectivePGP/ObjectivePGP.m:766
2017-02-22 13:46:28.107 pass[4066:71397] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Empty data'
```

Also the application will continue to crash immediately after a restart

**Comment by mssun** [comment]
Fixed by commit: e1f3a08

**Comment by mssun** [comment]
I am closing this issue. Free feel to reopen if this is not thoroughly fixed.
